tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -19,12 +20,14 @@ def mock_openpbs(pytestconfig, monkeypatch, tmp_path):
     mock_bin(monkeypatch, tmp_path)
 
 
-@pytest.mark.timeout(30)
 @pytest.mark.integration_test
 @pytest.mark.usefixtures("copy_poly_case")
 def test_openpbs_driver_with_poly_example():
+    queue_name = os.getenv("_ERT_TESTS_DEFAULT_QUEUE_NAME")
     with open("poly.ert", mode="a+", encoding="utf-8") as f:
-        f.write("QUEUE_SYSTEM TORQUE\nNUM_REALIZATIONS 2")
+        f.write("QUEUE_SYSTEM TORQUE\nNUM_REALIZATIONS 2\n")
+        if queue_name is not None:
```

Comment:
```suggestion
        if (queue_name: = os.getenv("_ERT_TESTS_DEFAULT_QUEUE_NAME")) is not None:
```

Reply:
Get one less line also for the other tests and queue_name is used only in the if statement.

Reply:
fixing!

Reply:
used a fixture as well.